src/app/shared/components/combobox/component.ts

```diff
@@ -144,5 +144,6 @@ export class FilteredComboboxComponent extends BaseFormValidator implements OnIn
 
   writeValue(value: string | string[]) {
     this.form.get(ComboboxControls.Select).setValue(value, {emitEvent: false});
+    this.selected = value;
```

Comment:
Reason:
If `this.selected` is not passed on from parent component then when `ngOnChanges` method is called (due to `options` Input being changed) it overrides the selected value because of:
```
    this.form.get(ComboboxControls.Select).setValue(this.selected);
```
`writeValue` is called when value of this form control is updated in the parent component. So by updating `this.selected` here we make sure that the value is not overridden.